### PR TITLE
Alphabetize the credits page

### LINKED ISSUE #3813: Credits page is no longer in Alphabetical Order
Since #3805 removed the last names from the credits page, the order that Scratch Team members are listed on the page doesn't make sense. They used to be ordered alphabetically by last name, but now since the last names are removed, they appear to be in a random order.

They should be reordered to be alphabetical by first name so that the page makes more sense.

### Steps to Reproduce

1. Go to the [credits page](https://scratch.mit.edu/credits)
2. Observe that the Scratch Team members appear to be in a random order.

### Operating System and Browser

This would appear on any device and browser

### PR DESCRIPTION
**Pull request by Accio1**
Put the credits page in order by first names as opposed to last names which are no longer present.

### Resolves:
Resolves #3813 by putting the credits page in alphabetical order by first name instead of last name

### Changes:
Rearranges people.json so that it's in alphabetical order by first name

### CODE REVIEW

**Comment by Accio1** [comment]
@benjiwheeler

File: src/views/credits/people.json [hunk 1]
```diff
@@ -1,242 +1,242 @@
...
-        "name": "Yusuf"
+        "userName": "amylaser",
+        "userId": 17462181,
+        "name": "Amy"
     },
     {
-        "userName": "ceebee",
-        "userId": 2755634,
-        "name": "Christan"
+        "userName": "thisandagain",
```
Comment (R4356th) [inline src/views/credits/people.json:13]: Should his name be here? He left the Scratch Team.

Comment (benjiwheeler) [reply src/views/credits/people.json:13]: Thanks for asking -- checking, will get back to you

**Comment by benjiwheeler** [comment]
@Accio1 thanks, Andrew was working with the Scratch Team until recently, but he has indeed moved on. But for now you can keep him in the list, we can move him to past contributors in a separate change.

**Comment by Accio1** [comment]
> @Accio1 thanks, Andrew was working with the Scratch Team until recently, but he has indeed moved on. But for now you can keep him in the list, we can move him to past contributors in a separate change.

Okay, thanks for looking into it!

**Comment by benjiwheeler** [comment]
Looks good, thanks @Accio1!

**Pull request merged**